roles/container_runtime/templates/crio.conf.j2

```diff
@@ -115,6 +115,12 @@ pids_limit = 1024
 # Negative values indicate that no limit is imposed.
 log_size_max = 52428800
 
+# manage_network_ns_lifecycle determines whether we pin and remove network namespaces
+# and manage their lifecycle
```

Comment:
nit: can we put this clause inside of the if statement?

Reply:
Ah right, I should, that's a template.